packages/core/admin/admin/src/pages/MarketplacePage/utils/api.js

```diff
@@ -1,11 +1,19 @@
 import axios from 'axios';
 
 const fetchPlugins = async notify => {
-  const { data } = await axios.get(`${process.env.STRAPI_ADMIN_MARKETPLACE_API_URL}/plugins`);
+  const { data: response } = await axios.get(
+    `${process.env.STRAPI_ADMIN_MARKETPLACE_API_URL}/plugins`
```

Comment:
Why do you use an env variable here?

I am a bit concerned about generated app with this, we don't create an `.env` file with the correct env variable so you'll need to add to the `core/admin/env.js` file so it will be exposed anyway.

I'd recomment to simply hardcode the endpoint.

Reply:
I wish we could avoid hardcoding, because we'll need to change that URL in development in the future. For example, when we'll work on adding filters with API params to the API, we'll run it on localhost to test it before deployment.

Regarding the .env file, on the previous PR (#12525), [we set a default value](https://github.com/strapi/strapi/blob/strapi-market/packages/core/admin/env.js#L28-L29) on the env.js file, so that we could still override it by manually creating the env file.

Do you think that's okay?

Reply:
It will still require a full rebuild, I am not sure sure it's needed to use env variable here. I think it's best to keep that URL in the file where it is used as this is not the purpose of env variables.

If we need to add filter later we need to ask our users to rebuild their admin panel entirely and not to go their `.env` file to add the new params to the marketplace url, it should be transparent for them.

Can you make an update by removing from the `.env` file?

Reply:
How about, inside this `MarketplacePage/utils/api.js`:

```js
const marketplaceApiUrl = process.env.STRAPI_ADMIN_MARKETPLACE_API_URL || 'https://market-api.strapi.io'
```

This way we don't touch the env.js file, we hardcode the default value for everyone, but we can still override it for dev purposes

Reply:
I don't like adding complexity when it's not needed let's make the change when we need it, right now it's a bit too early.